### PR TITLE
Update Korebuild

### PR DESCRIPTION
**Pull request by rynowak**
Related to #870

### CODE REVIEW

File: korebuild.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
-  "channel": "dev",
+  "channel": "release/2.1",
   "toolsets": {
     "visualstudio": {
       "required": false,
       "includePrerelease": true,
-      "minVersion": "15.0.26730.03",
+      "minVersion": "[15.0.26730.03, 15.8)",
```
Comment (rynowak) [inline korebuild.json]: We need to put an upper bound on which versions of MSBuild we use to build the VSIX. The current preview of VS isn't compatible with this SDK for building **BUILDING THE VSIX AT THE COMMANDLINE ONLY**.

I'm going to check on this before merging this PR and see if there's another option. This is the setup that razor is using currently.

File: korebuild.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
-  "channel": "dev",
+  "channel": "release/2.1",
```
Comment (rynowak) [inline korebuild.json]: Using the release version of the tools, which is going to track 2.1 and its patches. If we retarget at later builds, we can update then. I figured we don't want to use 2.2 yet.

**Comment by NTaylorMullen** [comment]
> Blazor is using an ancient version of Korebuild, but the new ones in the release channel are incompatible with desktop MSBuild in 15.8p1. Ideas?

When you say incompatible what error are you getting?

**Comment by rynowak** [comment]
```
C:\Users\Ryan\.dotnet\x64\sdk\2.1.300-rtm-008823\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.ta
rgets(199,7): error MSB4064: The "ProjectAssetsCacheFile" parameter is not supported by the "ResolvePackageAssets" task.
 Verify the parameter exists on the task, and it is a settable public instance property. [F:\git\aspnet\Blazor\tooling\M
icrosoft.VisualStudio.LanguageServices.Blazor\Microsoft.VisualStudio.LanguageServices.Blazor.csproj] [C:\Users\Ryan\.dot
net\buildtools\korebuild\2.1.0-rtm-15789\KoreBuild.proj]
```

**Comment by NTaylorMullen** [comment]
@rynowak remove the `/m;` from the msbuild item group in both locations in your VSIX.targets. Like in [this](https://github.com/aspnet/Razor/commit/63af9221dc147c4931449e6ae37f27f39d314f1b#diff-ac80d3760c2af1ba56c6c1aa46d74fa4) commit.

**Comment by rynowak** [comment]
Awesome! That did the trick.

**Pull request merged**